setup.py

```diff
@@ -12,8 +12,10 @@
 import re
 import sys
 import numpy as np
-from scipy._build_utils import numpy_nodepr_api
```

Comment:
Can you check this with 
- scipy>=1.3.1,<=1.7.3
and see you can get scipy._build_utils